executor/common_kvm_amd64_syzos.h

```diff
@@ -71,19 +75,37 @@ static void guest_handle_wr_drn(struct api_call_2* cmd);
 static void guest_handle_in_dx(struct api_call_2* cmd);
 static void guest_handle_out_dx(struct api_call_3* cmd);
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+void dummy_null_handler(void);
```

Comment:
I think this can be just:
```
__attribute__((naked)) GUEST_CODE static void dummy_null_handler()
{
	asm("iretq");
}
```

Reply:
For what it's worth, `naked` does not allow non-assembly contents, but that's not a big problem.